pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<!-- 
+Thanks for opening a pull request and helping Enarx grow.
+Please remember to
+- mention any issue this PR closes using its number #123 or enarx/repo-name#123, cf.
+  [documentation](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests)
```

Comment:
We want more than this: specifically, if this PR closes an issue, it should be linked to that issue with a closing keyword. Just linking to the issue isn't strong enough; it must have a keyword in front of it.

See [here](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for documentation on it.

Reply:
Ah good point. Let me clarify this, thanks. Also, thanks for the link to the documentation.